README.md

```diff
@@ -295,7 +296,9 @@ To configure the package behaviour, you can modify your `network_params.json` fi
       "mev_flood_seconds_per_bundle": 15,
       // A custom flood script that increases the balance of the coinbase addresss leading to more reliable
```

Comment:
`// Optional parameters to send to the custom_flood script that sends reliable payloads
`